doc/menu_lua_api.md

```diff
@@ -340,6 +341,12 @@ Package - content which is downloadable from the content db, may or may not be i
           error_message = "",  -- message or nil
       }
       ```
+* core.get_content_translation(path, domain, string)
+  * Translates `string` using `domain` in content directory at `path`.
+  * Textdomains will be found by looking through all locale folders.
+  * String should contain translation markup from `core.translate(textdomain, ...)`.
+  * Ex: `core.get_content_translation("mods/mymod", "mymod", core.translate("mymod", "Hello World"))` will translate
```

Comment:
Longer than 95 characters. Needs breaking.

Reply:
I've broken "will translate" to the next line, but breaking the inline code snippet would import readability